Add status command

sergiobuj opened the pull request:
closes #1471 by adding the status command that prints the state of a
machine.

Signed-off-by: Sergio Botero <sergiobuj@gmail.com>

hairyhenderson commented on commands/commands.go:
```diff
@@ -380,6 +380,12 @@ var Commands = []cli.Command{
 	{
+		Name:        "status",
+		Usage:       "Get the status of a machine",
+		Description: "Argument(s) are one or more machine names.",
```
This implies it can handle multiple machines, but the purpose of the command is to report on _one_ machine.

hairyhenderson commented:
Thanks for the PR, @sergiobuj! Just the one issue. Also we'll need to add some docs for the command.

sergiobuj commented:
Hi,
I just added the info in the docs and changed the description. Please let me know if that looks ok so I can squash the commits and sign them off.

hairyhenderson commented:
:+1:

/cc @nathanleclaire @ehazlett

nathanleclaire commented on commands/status.go:
```diff
@@ -0,0 +1,18 @@
+	if err != nil {
+		log.Errorf("error getting state for host %s: %s", host.Name, err)
+	}
+	fmt.Println(currentState)
```
Please use `log.Info` here, for consistency's sake (it should be the same output).

nathanleclaire commented:
Looks pretty good.  Please add a few small integration test, probably to the core test at `test/integration/core/core-commands.bats`, which exercise this feature.  Come to think of it, you could probably just change:

```
@test "$DRIVER: machine should show stopped after stop" {
  run machine ls
  [ "$status" -eq 0  ]
  [[ ${lines[1]} == *"Stopped"*  ]]
}
```

to use `machine status` instead of the `${lines[1]}` check.

sergiobuj commented:
Hi @nathanleclaire, I changed the logging and added the integration test (sorry I missed it).

Please let me know if I am missing anything else. Thanks!

nathanleclaire commented:
LGTM

I'm going to have to hold off on merging and ask you to rebase until after https://github.com/docker/machine/pull/1490 is merged, though (it will be soon).

sergiobuj commented:
Ok, let's do that.

nathanleclaire commented:
@sergiobuj Thanks, I will ping you when ready!

nathanleclaire commented:
@sergiobuj OK, please rebase to the relevant section. If you need any help let me know.

sergiobuj commented:
@nathanleclaire done, let me know how that looks.

ehazlett commented:
LGTM
